app/informativedialog.h

```diff
@@ -0,0 +1,23 @@
+#ifndef INFORMATIVEDIALOG_H
+#define INFORMATIVEDIALOG_H
+
+#include <QDialog>
+
+namespace Ui
+{
+class InformativeDialog;
+}
+
+class InformativeDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit InformativeDialog(QString title, QString description = QString(), QWidget *parent = 0 );
+    ~InformativeDialog();
+
+private:
+    Ui::InformativeDialog *ui;
+};
```

Comment:
It's not quite the same, as I mentioned in my first reply, the button is centred and it doesn't have the icon. You can't control those things as far as I know with the standard QMessageBox.

Ideally I just want the text aligned with the icon if I were to use the standard QMessageBox::Information, but even that I can't do.